django/thunderstore/core/utils.py

```diff
@@ -23,3 +32,42 @@ def get_serializer_context(self):
         context = super().get_serializer_context()
         context["community_site"] = self.request.community_site
         return context
+
+
+def capture_exception(exception: Exception) -> None:
+    """Raises exception when running tests or NO_SILENT_FAIL."""
+    testing = "PYTEST_CURRENT_TEST" in os.environ
+    if testing or settings.NO_SILENT_FAIL:
+        raise exception
+    if not testing and settings.SENTRY_DSN:
```

Comment:
I don't think you need to check for `SENTRY_DSN`, and can just call the `capture_sentry_exception` even when it's not configured. Correct me if I'm wrong.

Reply:
Is there a possible disadvantage of checking?

Reply:
Let's say we change the sentry to derive it's configuration via some other channel than the `SENTRY_DSN`, this check now has a direct dependency on it being used via `SENTRY_DSN`. So not a very large issue, but it's code that doesn't have to exist